IsraelHiking.API/Services/NonApiMiddleware.cs

```diff
@@ -115,52 +121,9 @@ private Task Write(HttpContext context, string text)
             return context.Response.WriteAsync(text);
         }
 
-        /// <summary>
-        /// This method is used to create a page with information for crawlers
-        /// </summary>
-        /// <param name="title"></param>
-        /// <param name="thumbnailUrl"></param>
-        /// <param name="description"></param>
-        /// <returns></returns>
-        public static string GetPage(string title, string thumbnailUrl, string description)
+        private Task WriteHomePage(HttpContext context, string title, string thumbnailUrl, string description, string language="")
```

Comment:
This function is a one liner, not sure it's useful as a one liner, I would consider removing it and using the internal function instead when this function is called.